cli/multisig.go

```diff
@@ -89,7 +89,7 @@ var msigCreateCmd = &cli.Command{
 		},
 	},
 	Action: func(cctx *cli.Context) error {
-		if cctx.NArg() < 1 {
+		if cctx.NArg() != 1 {
```

Comment:
`ArgsUsage` has a `...` then later we do a `for _, a := range cctx.Args().Slice() {`